Plan/common/src/main/java/com/djrapitops/plan/storage/database/SQLDB.java

```diff
@@ -58,28 +63,37 @@
  */
 public abstract class SQLDB extends AbstractDatabase {
```

Comment:
![image](https://user-images.githubusercontent.com/20226336/139555449-60e1fe72-6d7b-40c4-a99d-e1ffe2fa4fbf.png)
Ran into this after adding download code, what do you think would be the best way to avoid this?

Reply:
Increase The max complexity to 70 in the checkstyle.xml in the /Plan/config folder